bin/millau/node/src/chain_spec.rs

```diff
@@ -158,6 +158,8 @@ fn testnet_genesis(
 		pallet_substrate_bridge: Some(BridgeRialtoConfig {
 			// We'll initialize the pallet with a dispatchable instead.
 			init_data: None,
+			owner: None,
```

Comment:
Imo it should be `root_key`. But it may be changed in follow up PR. We would need to switch both MillauInRialto and RialtoInMillau pallets to start uninitialized with owner set to //Alice (root_key). Then they should be initialized with #483

Reply:
We do include `sudo` pallet as well, so we can always issue as root, but I agree it would be simpler to set it to `Some(root_key)`

Reply:
I'm not sure I understand the rational of setting the owner to be the root key in this case. Both Root and the Owner have the same level of control in the pallet, and if you want the owner to be Root you can achieve this result by just not setting an Owner

Reply:
`root_key` is not `Root` account/origin. It's the same key that can control `sudo` afaict, so it should be Alice's key.

Reply:
So from what I understand the `root_key` can be any account. In our case the `root_key` is used as the Sudo (although it doesn't have to be). Since this is the case isn't it still redundant to set `owner` to the `root_key` since the `root_key` is already `Sudo`?